### PR TITLE
Makes some small improvements to the UI

### PR DESCRIPTION
**Pull request by orowith2os**
# Description
I fixed some typos, and improved where I thought it could be.

Fixes #(issue)
Doesn't fix an issue, but improves the user experience.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. 
- I have not tested my changes yet.

### CODE REVIEW

File: bottles/frontend/ui/details-bottle.ui [hunk 1]
```diff
@@ -118,12 +118,12 @@
                         <property name="valign">center</property>
                         <property name="halign">center</property>
                         <property name="margin-bottom">5</property>
-                        <property name="text" translatable="yes">Run with Arguments…</property>
+                        <property name="text" translatable="yes">Run with addition arguments…</property>
```
Comment (TheEvilSkeleton) [inline bottles/frontend/ui/details-bottle.ui]:
```suggestion
                        <property name="text" translatable="yes">Run with additional arguments…</property>
```

File: bottles/frontend/ui/details-bottle.ui [hunk 2]
```diff
@@ -118,12 +118,12 @@
...
                     </object>
                 </child>
                 <child>
                     <object class="GtkCheckButton" id="check_terminal">
-                        <property name="label" translatable="yes">Launch in Terminal…</property>
+                        <property name="label" translatable="yes">Launch in a terminal…</property>
```
Comment (TheEvilSkeleton) [inline bottles/frontend/ui/details-bottle.ui:126]:
```suggestion
                        <property name="label" translatable="yes">Launch in a Terminal…</property>
```

File: bottles/frontend/ui/details-bottle.ui (hunk 1 above)
Comment (axtloss) [inline bottles/frontend/ui/details-bottle.ui:121]: Shouldn't this be `Run with additional arguments`?

File: bottles/frontend/ui/details-preferences.ui [hunk 3]
```diff
@@ -21,7 +21,7 @@
                     <object class="AdwActionRow" id="row_dxvk">
                         <property name="activatable-widget">switch_dxvk</property>
                         <property name="title" translatable="yes">DXVK</property>
-                        <property name="subtitle" translatable="yes">Improves performance of DirectX 11 games and 3D applications.</property>
+                        <property name="subtitle" translatable="yes">Improves performance of DirectX 9/10/11 games and 3D applications by translating their API calls to the Vulkan equivalents.</property>
```
Comment (TheEvilSkeleton) [inline bottles/frontend/ui/details-preferences.ui:24]:
```suggestion
                        <property name="subtitle" translatable="yes">Improves performance of DirectX 9/10/11 games and 3D applications by translating to Vulkan.</property>
```

File: bottles/frontend/ui/details-preferences.ui [hunk 4]
```diff
@@ -40,7 +40,7 @@
                     <object class="AdwActionRow" id="row_vkd3d">
                         <property name="activatable-widget">switch_vkd3d</property>
                         <property name="title" translatable="yes">VKD3D</property>
-                        <property name="subtitle" translatable="yes">Improves performance of DirectX 12 games and 3D applications.</property>
+                        <property name="subtitle" translatable="yes">Improves performance of DirectX 12 games and 3D applications by translating their API calls to the Vulkan equivalents.</property>
```
Comment (TheEvilSkeleton) [inline bottles/frontend/ui/details-preferences.ui:43]:
```suggestion
                        <property name="subtitle" translatable="yes">Improves performance of DirectX 12 games and 3D applications by translating to Vulkan.</property>
```

File: bottles/frontend/ui/details-bottle.ui (hunk 1 above)
Comment (orowith2os) [reply bottles/frontend/ui/details-bottle.ui:121]: Yes it should, I'll fix the typo.

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]

File: bottles/frontend/ui/details-bottle.ui (hunk 1 above)
Comment (TheEvilSkeleton) [reply bottles/frontend/ui/details-bottle.ui]: Looks like this one was fixed.

File: bottles/frontend/ui/details-bottle.ui (hunk 2 above)
Comment (orowith2os) [reply bottles/frontend/ui/details-bottle.ui:126]: Why the capitalization? I don't see why it would be needed.

File: bottles/frontend/ui/details-preferences.ui [hunk 5]
```diff
@@ -59,7 +59,7 @@
                     <object class="AdwActionRow" id="row_nvapi">
                         <property name="activatable-widget">switch_nvapi</property>
                         <property name="title" translatable="yes">DLSS (DXVK-NVAPI)</property>
-                        <property name="subtitle" translatable="yes">Provide DLSS support if available and Nvidia's NVAPI.</property>
+                        <property name="subtitle" translatable="yes">Provide DLSS support if available, and Nvidia's NVAPI.</property>
```
Comment (axtloss) [inline bottles/frontend/ui/details-preferences.ui:62]: I would rephrase this to be `Provide DLSS and Nvidia NVAPI support if available`

File: bottles/frontend/ui/details-preferences.ui [hunk 6]
```diff
@@ -509,7 +509,7 @@
                         <property name="subtitle" translatable="yes">Use a restricted/managed environment for this bottle.</property>
                         <child>
                             <object class="GtkButton" id="btn_manage_sandbox">
-                                <property name="tooltip-text" translatable="yes">Manage Sandbox permissions</property>
+                                <property name="tooltip-text" translatable="yes">Manage the sandbox permissions</property>
```
Comment (axtloss) [inline bottles/frontend/ui/details-preferences.ui:512]: I think the original `Manage sandbox permissions`  was already correct

File: bottles/frontend/ui/details-preferences.ui [hunk 7]
```diff
@@ -754,7 +754,7 @@
                     <object class="AdwActionRow">
                         <property name="activatable-widget">switch_obsvkc</property>
                         <property name="title" translatable="yes">OBS Vulkan Capture</property>
-                        <property name="subtitle" translatable="yes">Toggle OBS game capture for all programs.</property>
+                        <property name="subtitle" translatable="yes">Toggle OBS game capture for all Vulkan programs.</property>
```
Comment (axtloss) [inline bottles/frontend/ui/details-preferences.ui:757]:
```suggestion
                        <property name="subtitle" translatable="yes">Toggle OBS game capture for all programs.</property>
```
I'm pretty sure the obs-vkcapture isn't only for vulkan

File: bottles/frontend/ui/details-preferences.ui (hunk 6 above)
Comment (orowith2os) [reply bottles/frontend/ui/details-preferences.ui:512]: it was, but I think it would be best to have it as `Manage the sandbox permissions`. I can revert the change, if you'd like.

File: bottles/frontend/ui/details-preferences.ui (hunk 7 above)
Comment (TheEvilSkeleton) [reply bottles/frontend/ui/details-preferences.ui:757]: You're right, it's for OpenGL too.

File: bottles/frontend/ui/details-preferences.ui (hunk 6 above)
Comment (axtloss) [reply bottles/frontend/ui/details-preferences.ui:512]: I think `Manage the sandbox permissions` isn't grammatically correct, I don't think that there should be a `the` in front of the `sandbox`

File: bottles/frontend/ui/details-preferences.ui (hunk 7 above)
Comment (orowith2os) [reply bottles/frontend/ui/details-preferences.ui:757]: Oh yes, that is correct. Based on [this repo](https://github.com/nowrep/obs-vkcapture), it functions for OpenGL applications as well.

File: bottles/frontend/ui/details-bottle.ui (hunk 2 above)
Comment (TheEvilSkeleton) [reply bottles/frontend/ui/details-bottle.ui:126]: It's to keep it consistent with the dropdown menu and conforming with the GNOME HIG.

File: bottles/frontend/ui/details-preferences.ui (hunk 6 above)
Comment (mirkobrombin) [reply bottles/frontend/ui/details-preferences.ui:512]: GNOME Circle asked to Capitalize All Titles

Comment (orowith2os) [reply bottles/frontend/ui/details-preferences.ui:512]: Got some input from an external person, I'll capitalize it to be in accordance with the HIG and for proper grammar.

Comment (TheEvilSkeleton) [reply bottles/frontend/ui/details-preferences.ui:512]: Except some words like "a", "the", etc.

**Review by axtloss — APPROVED** [review]

File: bottles/frontend/ui/details-bottle.ui [hunk 8]
```diff
@@ -118,12 +118,12 @@
                         <property name="valign">center</property>
                         <property name="halign">center</property>
                         <property name="margin-bottom">5</property>
-                        <property name="text" translatable="yes">Run with Arguments…</property>
+                        <property name="text" translatable="yes">Run with additional arguments…</property>
```
Comment (TheEvilSkeleton) [inline bottles/frontend/ui/details-bottle.ui:121]:
```suggestion
                        <property name="text" translatable="yes">Run with Additional Arguments…</property>
```

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]

**Review by TheEvilSkeleton — APPROVED** [review]

File: bottles/frontend/ui/about.ui [hunk 9]
```diff
@@ -20,6 +20,7 @@ BrainBlasted https://github.com/BrainBlasted
 Francesco Masala &lt;mail@francescomasala.me&gt;
 Hari Rana (TheEvilSkeleton) https://theevilskeleton.gitlab.io/
 axtlos https://axtloss.github.io/</property>
+Oro https://github.com/orowith2os
```
Comment (TheEvilSkeleton) [inline bottles/frontend/ui/about.ui:23]:
```suggestion
axtlos https://axtloss.github.io/
Oro https://github.com/orowith2os</property>
```

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]
Should be like this no?

File: bottles/frontend/ui/about.ui (hunk 9 above)
Comment (orowith2os) [reply bottles/frontend/ui/about.ui:23]: Hm, but there are other people below that with no `</property>` tag.

Comment (axtloss) [reply bottles/frontend/ui/about.ui:23]: The other people are in a different property, https://github.com/bottlesdevs/Bottles/blob/79d03062701d52847941fae86453ac28899b6dd4/bottles/frontend/ui/about.ui#L24
The way you did it you'd be in no property and it should fail to build properly

Comment (orowith2os) [reply bottles/frontend/ui/about.ui:23]: Ah, alright then. That makes sense.

**Review by TheEvilSkeleton — APPROVED** [review]
